lang/cs/Org.Apache.REEF.Client/AzureBatch/AzureBatchREEFDotNetParamSerializer.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+// 
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+using System.IO;
+using Org.Apache.REEF.Client.Avro.AzureBatch;
+using Org.Apache.REEF.Common.Avro;
+using Org.Apache.REEF.Common.Files;
+using Org.Apache.REEF.Tang.Annotations;
+
+namespace Org.Apache.REEF.Client.AzureBatch
+{
+    /// <summary>
+    /// Job/application parameter file serializer for <see cref="AzureBatchDotNetClient"/>.
+    /// </summary>
+    internal sealed class AzureBatchREEFDotNetParamSerializer
```

Comment:
This is a very specific class for having almost no functionality. I would suggest removing if possible or making a SerializerUtil.

Reply:
I will remove SerializeJobArgsToBytes methods. 

This class will not be able to share with other runtime. In YARN, it requires both AppSubmissionParameters and JobParameters. In AzureBatch, only JobParameters is needed to be serialized.

Reply:
Removed it.